src/java/mc/alk/arena/listeners/competition/DamageListener.java

```diff
@@ -111,7 +113,8 @@ public void onEntityDamageEvent(EntityDamageEvent event) {
 			if (t != null && t.hasMember(target)){ /// attacker is on the same team
 				event.setCancelled(true);
 			} else {/// different teams... lets make sure they can actually hit
-				event.setCancelled(false);
+			    if (!HeroesController.enabled())
```

Comment:
What's the purpose of this code? I'm not really sure why you put this Heroes check here...

Reply:
This is to address "Problem 2" in my issue post ( https://github.com/BattlePlugins/BattleArena/issues/105 )

Basically, Heroes will cancel all damage events to make someone "invulnerable" or "stunned" for a short period of time. The problem with this piece of code, is that it is occuring _after_ heroes cancels the event. BattleArena will then re-enable the event, and prevent our invulnerability / stun code from working within arenas.

Reply:
Oh okay, that makes more sense. Thanks!